src/modules/file-upload-input.marko

```diff
@@ -0,0 +1,88 @@
+<div id="file-upload-input">
+    <section-header metadata=input.metadata name=input.name alpha="true"/>
+
+    <p>
+        The file-upload-input allows users to upload one or more files to a location.
```

Comment:
I like the intro here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

I think we could just copy or paraphrase that, e.g.

The file upload input lets the user choose one or more files from their device storage. Once chosen, the files can be uploaded to a server using [form submission](https://developer.mozilla.org/en-US/docs/Learn/Forms), or manipulated using JavaScript code and [the File API](https://developer.mozilla.org/en-US/docs/Web/API/File_API/Using_files_from_web_applications).

Reply:
I would also add a second paragraph which can replace the Drag and Drop section that you have:

The entire element can function as a drop zone for drag and drop, however this functionality requires JavaScript and is outside the scope of a CSS library such as eBay Skin.

Reply:
Thanks! Updated